tutorials/plot_background_morph.py

```diff
@@ -0,0 +1,875 @@
+possible to construct a composite blurring matrix
+
+.. math::    S^{(N)} = \prod_{p = 1}^N {S^{(p)}}\ .
+
+However, it turns out to be computationally more effective
+to do blurring with an iteration. The above formula for :math:`S^{(N)}` also
+shows that the smudging (smoothing) operation is linear.
+
+.. _tut_surface_morphing_precompute:
+
+Precomputing
+------------
+
+The utility mne_make_morph_maps was
+created to assist mne_analyze and mne_make_movie in
+morphing. Since the morphing maps described above take a while to
+compute, it is beneficial to construct all necessary maps in advance
+before using mne_make_movie .
+The precomputed morphing maps are located in ``$SUBJECTS_DIR/morph-maps`` .
+mne_make_morph_maps creates this directory automatically if it does not exist.
+If this directory exists when mne_analyze or mne_make_movie is run
+and morphing is requested, the software first looks for already
+existing morphing maps there. Also, if mne_analyze or mne_make_movie have
+to recompute any morphing maps, they will be saved to
+``$SUBJECTS_DIR/morph-maps`` if this directory exists.
+
+The names of the files in ``$SUBJECTS_DIR/morph-maps`` are
+of the form:
+
+ <*A*> - <*B*> -``morph.fif`` ,
+
+where <*A*> and <*B*> are
+names of subjects. These files contain the maps for both hemispheres,
+and in both directions, *i.e.*, both :math:`M^{(AB)}` and :math:`M^{(BA)}`, as
+defined above. Thus the files <*A*> - <*B*> -``morph.fif`` or <*B*> - <*A*> -
+``morph.fif`` are functionally equivalent. The name of the file produced by
+mne_analyze or mne_make_movie depends on the role of <*A*> and <*B*> in
+the analysis.
+
+If you choose to compute the morphing maps in batch in advance,
+use :ref:`mne_make_morph_maps`.
+
+.. _tut_volumetric_morphing:
+
+Volumetric morphing
+===================
+
+The key difference between volumetric morphing and morphing surfaces, is
+that the data is represented as a volume in a gridded space. A volume is a
+3-dimensional data representation, but it is necessary to understand that the
+difference to a mesh (what's commonly referred to as "3D model") is that the
+mesh is an "empty" surface, while the volume is "filled". Whereas the mesh is
+defined by the vertices of the outer hull, the volume is defined by that and
+the points it is containing. Hence morphing volumetric data does not only
+require to map the surface data, but also it's content in the correct way.
+Note, that since volumetric data "contains" something, the number of points,
+when increasing resolution scales exponentially. A sphere of 5 mm radius, would
+contain :math:`4\pi r^2 \approx 314` vertices, when sampled a 1 mm resolution.
+The number of volume points however would scale with :math:`\frac{4}{3}\pi r^3`
+and thus at the same resolution would contain :math:`\approx 524` data points.
+Hence the 3D-volume representation of a sphere will always contain
+:math:`\frac{r}{3}` times more data points than the 3D mesh if represented in
+unit space.
```

Comment:
This is a nice mathematical fact, but is it relevant to understand the tutorial?

Reply:
yeah I know it's kinda unrelated - my goal was to get someone who maybe didn't think about this before just some grasp of what can be expected in terms of how different the data could be for both representations etc - removing?